psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -166,11 +166,6 @@ def scf_initialize(self):
     self.memory_jk_ = int(total_memory - collocation_memory)
     self.memory_collocation_ = int(collocation_memory)
 
-    # Print out initial docc/socc/etc data
-    if self.get_print():
-        core.print_out("  ==> Pre-Iterations <==\n\n")
-        self.print_preiterations()
-
     if self.get_print():
         core.print_out("  ==> Integral Setup <==\n\n")
 
```

Comment:
I guess what I am getting at here is that we can move `initialize_jk` to below all of the one-electron stuff. Where the one-electron stuff would become the `Pre-iterations` in total. `Integral Setup` would then become something specific to JK. I think it is useful to have some sort of information before JK is built which takes often takes more time than a few SCF iterations.

@psi4/editors This could use a look by everyone. 

@hokru Hits on a pretty important topic here! Apologies for convolving it with other issues, but this should be looked at holistically I think.

Reply:
No worries, I agree on carefully crafting how the output is ordered. Though I am not yet sure I fully understand where you want everything :-)  Let's see other people's comments.